### PR TITLE
What's new screen for software channels

### PR DESCRIPTION
**Pull request by irlaec**
image

### CODE REVIEW

**Comment by irlaec** [comment]
[What's new wizard --> Beta channel](https://trello.com/c/S4KmRX5Y/110-whats-new-wizard-beta-channel)

**Comment by andywerner** [comment]
Looks great!!!! 

About the wording:
- Let's find an official name for the channels and use it consistently. Eg:
  - Software Channel
  - Software Update Channel
- Make the red line blue; it's more on "info" level, not on 'warning' ;-)

File: octoprint_mrbeam/__init__.py [hunk 1]
```diff
@@ -365,7 +365,9 @@ def on_settings_save(self, data):
 			if "focusReminder" in data:
 				self._settings.set_boolean(["focusReminder"], data["focusReminder"])
 			if "dev" in data and "software_tier" in data['dev']:
+				self._logger.info("ANDYTEST on_settings_save software_tier: START")
```
Comment (andywerner) [inline octoprint_mrbeam/__init__.py:368]: Remove this

File: octoprint_mrbeam/__init__.py [hunk 2]
```diff
@@ -365,7 +365,9 @@ def on_settings_save(self, data):
...
 				switch_software_channel(self, data["dev"]["software_tier"])
+				self._logger.info("ANDYTEST on_settings_save software_tier: DONE")
```
Comment (andywerner) [inline octoprint_mrbeam/__init__.py:370]: Remove this

File: octoprint_mrbeam/software_update_information.py [hunk 3]
```diff
@@ -1,7 +1,6 @@
 
-import yaml
-import os
-import subprocess
+import threading
```
Comment (andywerner) [inline octoprint_mrbeam/software_update_information.py:2]: Remove this

File: octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2 [hunk 4]
```diff
@@ -1,9 +1,28 @@
...
+        <p>{{ _('Use the most stable software for your Mr Beam II. You will only get notified about officially released and well tested software updates.') }}</p>
+    </div>
+    <div class="control-group span6">
+        <h5><i class="fa fa-flask" aria-hidden="true"></i>{{ _('Beta Channel') }}: </h5>
+        <p>{{ _('Get the latest Mr Beam development! In this channel you can try the newest features even before they are officially released, with more frequent updates. Use this if you are curious, you can always switch back to the stable channel if you experience errors.') }}</p>
+    </div>
+</div>
+
+<div id="whatsnew_software_channel_choose">
+    <label for="whatsnew_software_channel_select"><strong>{{ _('Select a channel for your Mr Beam II:') }}</strong></label>
```
Comment (andywerner) [inline octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2:15]: please call it "software channel"

File: octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2 [hunk 5]
```diff
@@ -1,9 +1,28 @@
...
+    <select id="whatsnew_software_channel_select">
+        <option value="PROD" selected="selected">Stable</option>
+        <option value="BETA">Beta</option>
+        {% if env_local == 'DEV' %}
+            <option value="DEV">Develop</option>
+        {% endif %}
+    </select>
+    <div id="whatsnew_software_channel_switch">
+        <i class="fa fa-info" aria-hidden="true"></i>
+        {{ _('Switch between the different channels of the Mr Beam II software any time in the settings!') }}
```
Comment (andywerner) [inline octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2:25]: please call it "software channel"

Comment (andywerner) [inline octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2:25]: Please make this line appear in blue color

**Review by andywerner — CHANGES_REQUESTED** [review]

File: octoprint_mrbeam/__init__.py (hunk 1 above)
Comment (irlaec) [reply octoprint_mrbeam/__init__.py:368]: Fixed in f_channel_explanation_settings

File: octoprint_mrbeam/__init__.py (hunk 2 above)
Comment (irlaec) [reply octoprint_mrbeam/__init__.py:370]: Fixed in f_channel_explanation_settings

File: octoprint_mrbeam/software_update_information.py (hunk 3 above)
Comment (irlaec) [reply octoprint_mrbeam/software_update_information.py:2]: Fixed in f_channel_explanation_settings

File: octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2 (hunk 4 above)
Comment (irlaec) [reply octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2:15]: Fixed in f_channel_explanation_settings

File: octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2 (hunk 5 above)
Comment (irlaec) [reply octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2:25]: Fixed in f_channel_explanation_settings

Comment (irlaec) [reply octoprint_mrbeam/templates/wizard/wizard_whatsnew_0.jinja2:25]: Fixed in f_channel_explanation_settings

**Pull request closed without merging**